monai/bundle/config_item.py

```diff
@@ -210,7 +227,14 @@ class ConfigComponent(ConfigItem, Instantiable):
 
     """
 
-    non_arg_keys = {"_target_", "_disabled_", "_requires_", "_desc_", "_mode_"}
+    non_arg_keys = {
```

Comment:
So if you had a `StrEnum` object called `ConfigComponents`, you could have `non_arg_keys = set(ConfigComponents)` instead I believe.